app/styles/main.scss

```diff
@@ -146,3 +146,14 @@ p.subtext {
     margin-top: 90px;
   }
 }
+
+.visually-hidden {
+  border: 0;
+  clip: rect(0 0 0 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+}
```

Comment:
We want this to be focusable yet visually hidden but we want our content to be read by screen readers.